providers/dns/ipv64/ipv64.go

```diff
@@ -24,15 +24,13 @@ const (
 	EnvPropagationTimeout = envNamespace + "PROPAGATION_TIMEOUT"
 	EnvPollingInterval    = envNamespace + "POLLING_INTERVAL"
 	EnvHTTPTimeout        = envNamespace + "HTTP_TIMEOUT"
-	EnvSequenceInterval   = envNamespace + "SEQUENCE_INTERVAL"
 )
 
 // Config is used to configure the creation of the DNSProvider.
 type Config struct {
 	APIKey             string
 	PropagationTimeout time.Duration
 	PollingInterval    time.Duration
-	SequenceInterval   time.Duration
```

Comment:
Doesn't this change make it a breaking change? `Config` is exported, so removing fields could affect Lego-as-library users.

For now, I'd just add a comment and mark it for removal in v5, e.g.

```diff
-	SequenceInterval   time.Duration
+ 	SequenceInterval   time.Duration // Deprecated and not used. Will be removed in Lego v5.
```

Reply:
There is no breaking change because the option is not used and the option was introduced in v4.13.

Reply:
If we keep and deprecate the field `SequenceInterval` we also have to keep and deprecate the env var name (constant).

Reply:
Hm, true... So do we remove it (and deal with the fallout later, if there's any), or do we keep them with a deprecation note?

Reply:
We can keep them, it's better from the point of view of the project.